drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -1134,6 +1134,56 @@ static int bnxt_set_vf_vlan_filter_op(struct rte_eth_dev *dev, uint16_t vlan,
 	return rc;
 }
 
+static int bnxt_set_vf_rx_mode_op(struct rte_eth_dev *dev, uint16_t vf,
+				uint16_t rx_mask, uint8_t on)
+{
+	struct bnxt *bp = (struct bnxt *)dev->data->dev_private;
+	uint16_t flag = 0;
+	int rc;
+
+	if (!bp->pf.vf_info)
+		return -EINVAL;
+
+	if (vf >= bp->pdev->max_vfs)
+		return -EINVAL;
+
+	switch (rx_mask) {
```

Comment:
The name implies this is a set of bits, rather than an enumeration.  Should this be a series of if() statements instead of a switch?